begin-transaction.md

```diff
@@ -1,20 +1,19 @@
 ---
 title: BEGIN
-summary: Use the BEGIN statement to initiate a CockroachDB transaction.
+summary: Initiate a SQL transaction with the BEGIN statement in CockroachDB.
 toc: false
 ---
 
-The `BEGIN` [statement](sql-statements.html) initiates a [transaction](transactions.html), which bundles multiple SQL statements into a single all-or-nothing transaction. 
+The `BEGIN` [statement](sql-statements.html) initiates a [transaction](transactions.html), which either successfully executes all of the statements it contains or none at all.
+
+{{site.data.alerts.callout_info}}If the transaction contends for the same resources as other transactions and "loses", you can <a href="transactions.html#transaction-retries">retry it on the client side</a> using <a href="savepoint.html"><code>SAVEPOINT</code></a> (shown in the <a href="#examples">examples below</a>).{{site.data.alerts.end}}
```

Comment:
You can (and should) retry the transaction whether you use SAVEPOINT or not; SAVEPOINT makes the retries more efficient. 

We should have a section that explains optimistic vs pessimistic concurrency; I think links like this should point to that explanation and then we link from there to SAVEPOINT.